Source/ACE/Entity/GenericObject.cs

```diff
@@ -15,46 +15,24 @@ public GenericObject(AceObject aceObject)
         {
         }
 
-        public GenericObject(ObjectGuid guid, AceObject aceObject)
-            : base(guid, aceObject)
-        {
-        }
+        ////public GenericObject(ObjectGuid guid, AceObject aceObject)
```

Comment:
except, there are many WeenieType.Generic objects in the database.. its not going to be as useless as you are suggesting